src/gql/types/PhoneNumberType.php

```diff
@@ -0,0 +1,79 @@
+<?php
+/**
+ * @link https://www.ryanpascoe.co
+ * @copyright Copyright (c) Ryan Pascoe
+ * @license MIT
+ */
+
+namespace rynpsc\phonenumber\gql\types;
+
+use craft\gql\GqlEntityRegistry;
+use GraphQL\Type\Definition\ObjectType;
+use GraphQL\Type\Definition\Type;
+
+/**
+ * Class PhoneNumberType
+ *
+ * @author Jayden Smith
+ * @package Phone Number
+ * @since 1.4.0
+ */
+class PhoneNumberType
+{
+    /**
+     * @return string
+     */
+    public static function getName(): string
+    {
+        return 'phoneNumber_PhoneNumber';
```

Comment:
Is this the standard naming convention or would PhoneNumber be sufficient? Not too familiar with authoring GraphQL.

Reply:
@rynpsc generally `packageName_ModelName`